File: aspnet-core/src/ERP.Application.Shared/PayRoll/SlabSetup/Dtos/SlabSetupDto.cs
using System;
using Abp.Application.Services.Dto;

namespace ERP.Payroll.SlabSetup.Dtos
{
    public class SlabSetupDto : EntityDto
    {
        public int? TypeID { get; set; }

        public double? SlabFrom { get; set; }

        public double? SlabTo { get; set; }

        public double? Rate { get; set; }

        public double? Amount { get; set; }

        public bool Active { get; set; }

        public string AudtUser { get; set; }

        public DateTime? AudtDate { get; set; }

        public string CreatedBy { get; set; }

        public DateTime? CreateDate { get; set; }

        public string ModifiedBy { get; set; }

        public DateTime? ModifyDate { get; set; }

    }
}

File: angular/src/app/main/payroll/shared/dto/slabSetup-dto.ts
import { ICreateOrEditSlabSetupDto, IGetSlabSetupForEditOutput, IGetSlabSetupForViewDto, IPagedResultDtoOfGetSlabSetupForViewDto, ISlabSetupDto } from "../interface/taxSlabs-interface";

export class SlabSetupDto implements ISlabSetupDto {

    id: number;
    typeID: number;
    slabFrom: number;
    slabTo: number;
    rate: number;
    amount: number;
    active: boolean;
    audtUser: string;
    audtDate: Date;
    createdBy: String;
    createDate: Date;

}

export class GetSlabSetupForViewDto implements IGetSlabSetupForViewDto {
    slabSetup: SlabSetupDto;

}

export class CreateOrEditSlabSetupDto implements ICreateOrEditSlabSetupDto {
    id: number;
    typeID: number;
    slabFrom: number;
    slabTo: number;
    rate: number;
    amount: number;
    active: boolean;
    audtUser: string;
    audtDate: Date;
    createdBy: String;
    createDate: Date;
}

export class PagedResultDtoOfGetSlabSetupForViewDto implements IPagedResultDtoOfGetSlabSetupForViewDto {
    totalCount: number;
    items: GetSlabSetupForViewDto[];

}

export class GetSlabSetupForEditOutput implements IGetSlabSetupForEditOutput {
    slabSetup: CreateOrEditSlabSetupDto;

}